python/sync-with-algolia/src/utils.py

```diff
@@ -0,0 +1,24 @@
+import os, re
+
+__dirname = os.path.dirname(os.path.abspath(__file__))
+static_folder = os.path.join(__dirname, "../static")
+
+
+def get_static_file(file_name):
+    file_path = os.path.join(static_folder, file_name)
+    with open(file_path, "r") as file:
+        return file.read()
+
+
+def throw_if_missing(obj, keys):
```

Comment:
In other PR I noticed you did docs for utils file. Can we do that here as well, for all methods?

![Uploading CleanShot 2023-08-20 at 10.14.26@2x.png…]()